azure/azure-release-template.yml

```diff
@@ -94,7 +94,7 @@ extends:
           make publish-pacts > /dev/null
 
           # Poll deploying API until our deployed version matches the release version
-          url="https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH/_ping"
+          url="https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH/_status"
```

Comment:
The _status endpoint doesn't include the deployed commit, so I don't think this will work

Reply:
Think we need to poll _ping until we know the proxy deploy has happened, and then poll _status until the service has come up

Reply:
Ignore me I like jumping to conclusions